src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,93 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.DuplicateAnExistingLoginException;
+import core.basesyntax.exceptions.NotCorrectAgeException;
+import core.basesyntax.exceptions.NotCorrectPassword;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+
+    private static final User anna = new User();
+    private final RegistrationServiceImpl registrationService = new RegistrationServiceImpl();
+
+    @BeforeAll
+    static void beforeAll() {
+        anna.setAge(18);
+        anna.setId(15893234L);
+        anna.setLogin("Anna@gmaol.com");
+        anna.setPassword("123456789");
+    }
```

Comment:
```suggestion
    @BeforeAll
    static void beforeAll() {
        anna.setAge(18);
        anna.setLogin("Anna@gmaol.com");
        anna.setPassword("123456789");
    }
```

StorageDao sets id when save user to Stroage